pkg/commands/secretstoreentry/create.go

```diff
@@ -21,6 +22,24 @@ const (
 	maxSecretLen = maxSecretKiB * 1024
 )
 
+// The signing key is a public key that is used to sign client keys.
+// It's meant to be a long-lived key and infrequently (if ever) rotated.
+// Hardcoding it in the CLI gives us the benefit of distributing it via
+// a different channel from the client keys its signing.
+//
+// When we do rotate it, we will need to update this value and release a
+// new version of the CLI.  However, users can also override this with
+// the FASTLY_USE_API_SIGNING_KEY environment variable.
```

Comment:
@joeshaw where does `FASTLY_USE_API_SIGNING_KEY` get read via the CLI?

Reply:
I see that we check if the value is empty (below) but if the value is set then where/who is consuming that provided value?

Reply:
The value doesn't matter, that it's set to anything causes the check to be loosened.  This is mostly an escape hatch in case we rotate the signing key at some point in the future and a user has an old CLI version they can't upgrade for some reason, and are willing to take the very small risk of trusting the signing key provided via the API.

Reply:
Given the test failure, maybe this should be a command-line flag instead?